ibridges/irodsconnector/session.py

```diff
@@ -17,7 +17,7 @@ class Session:
     """
 
     def __init__(self, irods_env: Optional[dict] = None, irods_env_path: Optional[str] = None,
-                 password: Optional[str] = None):
+                 password: Optional[str] = None, irods_home: Optional[str] = None):
         """ iRODS authentication with Python client.
 
         Parameters
```

Comment:
Add hte parameter irods_home. Set to overwrite the "irods_home" value in your irods_environment.json.